tonic/src/transport/server/mod.rs

```diff
@@ -512,53 +526,226 @@ impl<L> Server<L> {
         let max_concurrent_streams = self.max_concurrent_streams;
         let timeout = self.timeout;
         let max_frame_size = self.max_frame_size;
+
         let http2_only = !self.accept_http1;
 
         let http2_keepalive_interval = self.http2_keepalive_interval;
         let http2_keepalive_timeout = self
             .http2_keepalive_timeout
             .unwrap_or_else(|| Duration::new(DEFAULT_HTTP2_KEEPALIVE_TIMEOUT_SECS, 0));
         let http2_adaptive_window = self.http2_adaptive_window;
-        let http2_max_pending_accept_reset_streams = self.http2_max_pending_accept_reset_streams;
 
-        let svc = self.service_builder.service(svc);
+        let http2_max_pending_accept_reset_streams = self.http2_max_pending_accept_reset_streams;
```

Comment:
The big chunk of chagnes around the server `Builder` struct are hard to retain in the original format because the API for configuring servers has changed subtly - instead of the `http2` prefixed methods, we need a temporary `Http2Builder`, which means we can't inline all of the confiugration calls into a single statement. Additioanlly, `http2_only` no longer accepts a `bool`, so can't be inlined either.

Reply:
Makes sense, thanks!